app/javascript/packs/vertical_nav.js

```diff
@@ -1,14 +1,19 @@
 import { VerticalNavWrapper as VerticalNav } from 'Navigation/components/VerticalNav'
 import { safeFromJsonString } from 'utilities/json-utils'
 
+const containerId = 'api_selector'
```

Comment:
Since there's another container already, I would rename this improve readability:
```suggestion
const apiSelectorId = 'api_selector'
```